Dockerfile

```
@@ -8,31 +8,39 @@ ENV DEBIAN_FRONTEND=noninteractive \
 
 RUN apt-get -qq update \
   && apt-get install -y --no-install-recommends \
-    apt-transport-https \
-    build-essential \
-    curl \
-    dirmngr \
-    gnupg \
-    git \
-    ruby-full \
-    twine \
+  apt-transport-https \
+  build-essential \
+  curl \
+  dirmngr \
+  gnupg \
+  git \
+  ruby-full \
+  twine \
+  jq \
+  unzip \
   && curl -fsSL https://packages.microsoft.com/config/debian/10/packages-microsoft-prod.deb -o /tmp/packages-microsoft-prod.deb \
   && dpkg -i /tmp/packages-microsoft-prod.deb \
   && rm /tmp/packages-microsoft-prod.deb \
   && echo 'deb [arch=amd64] https://download.docker.com/linux/debian buster stable' >> /etc/apt/sources.list \
   && curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add - \
   && apt-get update -qq \
   && apt-get install -y --no-install-recommends \
-    dotnet-sdk-3.1 \
-    docker-ce-cli \
+  dotnet-sdk-3.1 \
+  docker-ce-cli \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/* \
   && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s --  --profile minimal -y \
   && cargo --version \
   && cargo install cargo-hack \
   # Stick with 3.1.x as 3.2.x doesn't install on Debian Buster for some reason
   && gem update --no-document --system 3.1.5 \
-  && gem install cocoapods
+  && gem install cocoapods \
+  # Install https://github.com/getsentry/symbol-collector
+  && symbol_collector_url=$(curl -s https://api.github.com/repos/getsentry/symbol-collector/releases/tags/1.3.1 | \
```

@bruno-garcia you may want to use another version of the Symbol Collector.

---

I think we need to go with 1.2.1 for now (as you found, there's an issue since the .NET 5 upgrade on symbol collector native CLI builds:

```suggestion
  && symbol_collector_url=$(curl -s https://api.github.com/repos/getsentry/symbol-collector/releases/tags/1.2.1 | \
```